modular_skyrat/modules/aesthetics/guns/code/guns.dm

```diff
@@ -324,3 +324,166 @@
 	ammo_type = /obj/item/ammo_casing/a762
 	caliber = CALIBER_A762
 	max_ammo = 20
+	
+// BULLET UN-RETROFUTIRIZATION PR
```

Comment:
...Why un-retrofuturise SS13 of all things?

Reply:
Server objective.